frontend/src/views/MyApplications.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div class="my-application-forms">
+    <h1>내 지원서</h1>
+
+    <div
+      class="application-forms"
+      v-for="(applicationForm, index) in applicationForms"
+      :key="applicationForm.id"
+    >
+      <ApplicationFormItem
+        :recruitment="recruitments[index]"
+        :submitted="applicationForm.submitted"
+      />
+    </div>
+
+    <footer>
+      <a class="logo" href="#"></a>
+    </footer>
+  </div>
+</template>
+
+<script>
+import * as RecruitmentApi from "../api/recruitments"
+import * as ApplicationApi from "../api/applications"
+import ApplicationFormItem from "@/components/ApplicationFormItem"
+
+export default {
+  components: {
+    ApplicationFormItem,
+  },
+  data: () => ({
+    applicationForms: [],
+    recruitments: [],
+  }),
+  async mounted() {
+    const token = this.$store.state.token.value
+
+    try {
+      const { applicationFormsData } = await ApplicationApi.fetchMyApplicationForms(token)
+      const { recruitmentData } = await RecruitmentApi.fetchMyRecruitments(token)
```

Comment:
```suggestion
      const { data: applicationFormsData } = await ApplicationApi.fetchMyApplicationForms(token)
      const { data: recruitmentData } = await RecruitmentApi.fetchMyRecruitments(token)
```
axios가 리턴하는 객체에서 응답 body의 변수명은 항상 `data`입니다

Reply:
제가 명시하는것이 아니였군요 😅
아직 모르는게 많네요